rules/solid/tests/Rector/If_/ChangeAndIfToEarlyReturnRector/Fixture/nested_if_in_foreach.php.inc

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Rector\SOLID\Tests\Rector\If_\ChangeAndIfToEarlyReturnRector\Fixture;
+
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Name;
+
+class NestedIfInForeachClass
```

Comment:
this class gets transformed into the following which is no longer equivalent to the previous

```
<?php

namespace Rector\SOLID\Tests\Rector\If_\ChangeAndIfToEarlyReturnRector\Fixture;

use PhpParser\Node\Identifier;
use PhpParser\Node\Name;

class NestedIfInForeachClass
{
    private function refactorStmts(array $nodes): void
    {
        $g = [];

        foreach ($nodes as $node) {
            $x = ['A', 'B'];
            if (rand(1,2)) {
                // something
            } else {
                if (!$node instanceof Name) {
                    continue;
                }
                if (!$node instanceof Identifier) {
                    continue;
                }
                $x = [];
            }

            $g = $x;
        }
    }
}

?>
```

Reply:
What would be the correct transformation of `if`?

Reply:
I guess no tranformation shoud happen

Reply:
Agreed. The test fixture name should start with `skip_` and the class with `Skip`

Care to fix it at the same PR?

Reply:
Will try later

Reply:
Do you have suggestions on how we should fix this case / detect this case?

Reply:
Probably check the if + while combination